### PR TITLE
fix: suffix [shepherd] to not break semantic commit

### PR DESCRIPTION
**Pull request by DiegoRBaquero**

### CODE REVIEW

**Comment by aorinevo** [comment]
Taking a look.

**Comment by aorinevo** [comment]
Yea, looks good to me.  

This is loosely related to the branding open issue, in that it touches the same line of code (see https://github.com/NerdWalletOSS/shepherd/issues/145).

**Comment by nwalters512** [comment]
> This is loosely related to the branding open issue, in that it touches the same line of code

Good point - I'm honestly in favor of just dropping the "[shepherd]" label altogether - I think the footer Shepherd adds to PR messages is enough in the way of branding + identifying the PR as from Shepherd.

**Comment by aorinevo** [comment]
On second hand, can't we just drop `[shepherd]`?  Users can add the prefix to via `spec.title`.

File: src/adapters/git.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ abstract class GitAdapter implements IRepoAdapter {
   public async commitRepo(repo: IRepo): Promise<void> {
     const { migration: { spec } } = this.migrationContext;
     await this.git(repo).add('.');
-    await this.git(repo).commit(`[shepherd] ${spec.title}`);
+    await this.git(repo).commit(`${spec.title} [shepherd]`);
```
Comment (parshap) [inline src/adapters/git.ts:64]: Any opposition to removing the branding entirely?

```suggestion
    await this.git(repo).commit(spec.title);
```

At NerdWallet we still like to call out shepherd migrations in the title, but we can prefix `[shepherd]` manually in our spec titles.

Comment (nwalters512) [reply src/adapters/git.ts:64]: I think we've got consensus, let's do it!

**Comment by aorinevo** [comment]
See https://github.com/NerdWalletOSS/shepherd/blob/a849b16616a56dfc0a25907810899cc37db09bae/src/adapters/github.ts#L337

**Comment by parshap** [comment]
Good catch @aorinevo, we do that to determine whether it's safe to force-push over existing commits on a branch (we do it only if they are all Shepherd commits). Sounds like we'll need to keep `[shepherd]` in the title somewhere and update that logic to check for `[shepherd]` anywhere instead of just at the beginning. If that makes sense to everyone, @DiegoRBaquero can you update the PR to include that?

**Comment by DiegoRBaquero** [comment]
Updated :)

**Comment by parshap** [comment]
Looks good. We can look at how we could potentially remove the branding entirely as a separate effort from this PR.

Will wait for any comments from @aorinevo or @nwalters512 before merging.

**Comment by aorinevo** [comment]
That can be handled with the branding story by setting the default value to `[shepherd]`.  Consumer can override with empty string or their brand's name.

**Comment by aorinevo** [comment]
Danke @DiegoRBaquero!

**Pull request merged**